Modules/Forum/classes/class.ilForumExportGUI.php

```diff
@@ -374,6 +374,12 @@ public function exportHTML(): void
                 'thread_ids',
                 $this->refinery->kindlyTo()->listOf($this->refinery->kindlyTo()->int())
             );
+        } elseif($this->ref_id > 0) {
```

Comment:
Whitespace, again ^^. I again suggest to use the `Captain` to run the PHP-CS-FIXER on every commit :-).